### ISSUE TITLE
Edited Fee without XemCents

### ISSUE THREAD

**Issue by Espo02** [NemProject/NEMiOSApp#266]
The iOS App didn't accept x.y fee. Even if you want to sent with 100.3XEM fee it takes the default fee without a Message. The Android App can sent such Fee Transactions. 
If it is not possible to send on iOS x.yXEM then please make the error handling the same as if there isn't reached the minimun (correct the Fee to default and the user has to push the send button again).

![photo-2016-08-06-17-46-41_3001](https://cloud.githubusercontent.com/assets/17567386/17457677/1b465902-5bff-11e6-97aa-a91cf3e87754.PNG)
![photo-2016-08-06-17-46-44_3002](https://cloud.githubusercontent.com/assets/17567386/17457678/1e53baf4-5bff-11e6-8d5b-5297134ad4f0.PNG)
![photo-2016-08-06-17-46-58_3003](https://cloud.githubusercontent.com/assets/17567386/17457679/1fbc3f74-5bff-11e6-989e-1ae17ad77ee2.PNG)

Espo: NDEFC3-H4VAIU-FQ5GFT-IK3IJR-4DGKF7-PEQKRW-LPZ5

**Comment by jabo38** [comment]
that is strange that it rounds to the nearest whole number.

**Comment by Espo02** [comment]
That's not right. It canceld the user edited fee, sets it to the default min fee and executes the transaction immediately.

**Comment by jabo38** [comment]
Bounty Paid Transaction hash: 9b9b944e0815d177474c00cce006fbd70b9125c45a4e79097a7d369ee8200ea3
